src/GovUk.Education.ExploreEducationStatistics.Content.Model/DataSetFileMeta.cs

```diff
@@ -11,31 +12,52 @@ public class DataSetFileMeta
     public List<string> GeographicLevels { get; set; } = new();
 
     [JsonConverter(typeof(TimeIdentifierJsonConverter))]
-    public TimeIdentifier TimeIdentifier { get; set; }
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public TimeIdentifier? TimeIdentifier { get; set; } // EES-4918 to remove
+
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public List<int>? Years { get; set; } = new(); // EES-4918 to remove
 
-    public List<int> Years { get; set; } = new();
+    public TimePeriodRangeMeta TimePeriodRange { get; set; } = null!;
```

Comment:
Mentioned in our chat - we can add `required` across the model properties here and below and remove any unneeded default values.